docs/README.md

```diff
@@ -376,6 +377,47 @@ The `S3Bucket` class wraps an instance of `AmazonS3Client` (from AWS SDK for Jav
 
 The `SerializableAWSCredentials` class provides a serializable container for AWS credentials, extending the `AWSCredentials` class (from AWS SDK for Java).
 
+## Caching
+The `apso-caching` module provides provides utilities for caching. 
+
+To use it in an existing SBT project, add the following dependency to your `build.sbt`:
+
+```scala
+libraryDependencies += "com.velocidi" %% "apso-caching" % "@VERSION@"
+```
+
+Apso provides utilities to simplify the caching of method calls, with [ScalaCache](https://cb372.github.io/scalacache/)'s `Guava` or `Caffeine`. These are provided in an implicit scope, to make its invocation simpler:
```

Comment:
I would change that a little:
```suggestion
Apso provides utilities to simplify the caching of method calls, with [ScalaCache](https://cb372.github.io/scalacache/) 
and using either `Guava` or `Caffeine` as underlying cache implementations.

These utilities are provided as `.cached()` and `.cachedFn()` extension methods over all `FunctionN[]` types:
```

Reply:
Applied it on d258cbc8ba4b37f616816c0e7235134778fd07be.